docs/en/source/opcua-IoTIntegration.rst

```diff
@@ -0,0 +1,627 @@
+.. figure:: figures/OPC-UA.png
+
+
+Integrate Scorpio Broker with OPC UA IoT agent 
+****************************************************
+
+This is a step-by-step tutorial that will introduce in detail how to enable OPC UA to FIWARE connecting an OPC UA server to Context Broker using the agent. The OPC UA data will be automatically published in a FIWARE Context Broker using NGSI data model.
```

Comment:
Change the text as "This is a step-by-step tutorial to enable an OPC UA IOT Agent with Scorpio Broker Context Broker using the agent. The OPC UA data will be automatically published in a FIWARE Context Broker using NGSI-LD data model."

Reply:
Done